detox/src/devices/Fbsimctl.js

```diff
@@ -173,6 +173,14 @@ class Fbsimctl {
     await this._execFbsimctlCommand(options);
   }
 
+  async resetContentAndSettings(udid) {
```

Comment:
I don't think so. Need @rotemmiz to approve it, as he is the JS master around here. 😄